core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -211,11 +217,26 @@ public void getReport() {
   }
 
   @Test
-  public void getStoreMeta() {
+  public void getStoreMeta() throws Exception {
     mBlockWorker.getStoreMeta();
-    mBlockWorker.getStoreMetaFull();
     verify(mBlockStore, times(2)).getBlockStoreMeta(); // 1 is called at metrics registration
```

Comment:
No need to do in this PR, but will be better if you can do the similar changes against getStoreMeta just like what you did for `getStoreMetaFull`

Reply:
done